Replace SASS with Sass (#473)
springernature/frontend-playbook: css/house-style.md

CSS code (not compiled) should be linted with [styleLint](https://github.com/stylelint/stylelint). You can find _an example_ config file in the [config directory](config).

> Sass code should be linted with [sass-lint](https://github.com/sasstools/sass-lint) using the [Springer Nature

Reviewer: ```suggestion
Sass code should be linted with [Stylelint](https://stylelint.io/) using the [Springer Nature
```
We are using stylelint now!

> `sasslint-config`](https://github.com/springernature/sasslint-config-springernature). This configuration allows us to maintain consistency between different projects.

Reviewer: ```suggestion
`stylelint-config`](https://github.com/springernature/stylelint-config-springernature). This configuration allows us to maintain consistency between different projects.
```

You can check the [README](https://github.com/springernature/sasslint-config-springernature/blob/main/README.md) for details about installing and configuring the tools.

## Preprocessors

> Our preferred CSS preprocessor is [Sass using the SCSS syntax](how-we-write-css.md).

Reviewer 2: I reckon we should keep the link to Sass docs and, make the link to the md explicit.

```suggestion
Our preferred CSS preprocessor is [Sass](https://sass-lang.com) using the SCSS syntax. Find out about [how we write CSS](how-we-write-css.md).
```
